templates/partners/partial/_partner-filters/_programme-filters.html

```diff
@@ -1,24 +1,36 @@
 <label class="p-checkbox">
-  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="channel-reseller" aria-labelledby="channel-reseller">
+  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="channel-reseller" aria-labelledby="channel-reseller" />
   <span class="p-checkbox__label" id="channel-reseller">Channel / Reseller</span>
 </label>
 <label class="p-checkbox">
-  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="desktop" aria-labelledby="desktop">
+  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="desktop" aria-labelledby="desktop" />
   <span class="p-checkbox__label" id="desktop">Desktop</span>
 </label>
 <label class="p-checkbox">
-  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="global-system-integrators" aria-labelledby="global-system-integrators">
+  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="global-system-integrators" aria-labelledby="global-system-integrators" />
   <span class="p-checkbox__label" id="global-system-integrators">Global System Integrators</span>
 </label>
 <label class="p-checkbox">
-  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="ihv-oem" aria-labelledby="ihv-oem">
+  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="ihv-oem" aria-labelledby="ihv-oem" />
   <span class="p-checkbox__label" id="ihv-oem">IHV / OEM</span>
 </label>
 <label class="p-checkbox">
-  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="public-cloud" aria-labelledby="public-cloud">
+  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="public-cloud" aria-labelledby="public-cloud" />
   <span class="p-checkbox__label" id="public-cloud">Public Cloud</span>
 </label>
 <label class="p-checkbox">
-  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="silicon" aria-labelledby="silicon">
+  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="silicon" aria-labelledby="silicon" />
   <span class="p-checkbox__label" id="silicon">Silicon</span>
+</label>
+<label class="p-checkbox">
+  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="isv" aria-labelledby="isv" />
+  <span class="p-checkbox__label" id="isv">ISV</span>
+</label>
+<label class="p-checkbox">
+  <input type="checkbox" class="p-checkbox__input js-find-a-partner__filter" name="iot-devices" aria-labelledby="iot-devices" />
+  <span class="p-checkbox__label" id="iot-devices">IoT Devices</span>
```

Comment:
Looks like this value is not working, must be something else, since there are no items shown when you select this option